packages/infra/src/constructs/__snapshots__/lambdaDeployment.test.ts.snap

```diff
@@ -24,13 +24,24 @@ exports[`returns expected CloudFormation stack 1`] = `
     },
     "LambdaDeploymentCodeDeployLambdaDeploymentGroupC2B0BCFA": {
       "Properties": {
+        "AlarmConfiguration": {
+          "Alarms": [
+            {
+              "Name": {
+                "Ref": "LambdaDeploymentcodedeployalarm4FCB9B3A",
+              },
+            },
+          ],
+          "Enabled": true,
```

Comment:
Is there an option to toggle this enabled state, possibly via `actionsEnabled`? Just thinking whether deleting & recreating the entire alarm could be undesirable.

Reply:
Huh, what about just conditionally calling .addAlarm but always creating it?

Reply:
That seems reasonable 👍